google/cloud/security/inventory/inventory_loader.py

```diff
@@ -257,15 +298,27 @@ def main(argv):
          'status': ''},
     ]
 
+    if configs.get('inventory_groups'):
+        if _should_inventory_google_groups(configs):
```

Comment:
nit: the check is more about "can" instead of "should", so maybe change the name to _can_inventory_google_groups?